### ISSUE TITLE
working fork with pip install pydantic-to-typescript2

### ISSUE THREAD

**Issue by seanwessmith** [phillipdupuis/pydantic-to-typescript#46]
I created a fork with this merged so the community can easily install it via pip. The code is published to PyPI as pydantic-to-typescript2.

Phillip Dupuis is still included as an author.

Repository link: [Darius-Labs/pydantic-to-typescript2](https://github.com/Darius-Labs/pydantic-to-typescript2)

install pydantic-to-typescript2 via pip
```shell
pip install pydantic-to-typescript2
```

**Comment by joennlae** [comment]
Absolut legend :-)

**Comment by phillipdupuis** [comment]
Thank you for this! I was very busy for a bit there and honestly didn't even realize that this package had gained so much traction.

It should be resolved now that I've released v2 of `pydantic-to-typescript`. It supports all versions of pydantic (including v1 usage while v2 is installed) and we added a ton of tests to ensure nothing falls through the cracks. I'll definitely be inviting additional maintainers over the coming days.
